x-pack/plugins/ml/public/application/contexts/kibana/kibana_context.ts

```diff
@@ -30,33 +29,34 @@ import type { ContentManagementPublicStart } from '@kbn/content-management-plugi
 import type { SavedSearchPublicPluginStart } from '@kbn/saved-search-plugin/public';
 import type { PresentationUtilPluginStart } from '@kbn/presentation-util-plugin/public';
 import type { DataViewEditorStart } from '@kbn/data-view-editor-plugin/public';
+import type { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
+import type { UiActionsStart } from '@kbn/ui-actions-plugin/public';
 import type { MlServicesContext } from '../../app';
 
 interface StartPlugins {
+  cases?: CasesUiStart;
+  charts: ChartsPluginStart;
+  contentManagement: ContentManagementPublicStart;
+  dashboard: DashboardStart;
   data: DataPublicPluginStart;
   dataViewEditor: DataViewEditorStart;
   dataViews: DataViewsPublicPluginStart;
-  security?: SecurityPluginSetup;
-  licenseManagement?: LicenseManagementUIPluginSetup;
-  share: SharePluginStart;
-  embeddable: EmbeddableStart;
-  maps?: MapsStartApi;
-  triggersActionsUi?: TriggersAndActionsUIPublicPluginStart;
   dataVisualizer?: DataVisualizerPluginStart;
-  usageCollection?: UsageCollectionSetup;
-  fieldFormats: FieldFormatsRegistry;
-  dashboard: DashboardStart;
-  spacesApi?: SpacesPluginStart;
-  charts: ChartsPluginStart;
-  cases?: CasesUiStart;
-  unifiedSearch: UnifiedSearchPublicPluginStart;
-  core: CoreStart;
-  appName: string;
+  embeddable: EmbeddableStart;
+  fieldFormats: FieldFormatsStart;
   lens: LensPublicStart;
+  licenseManagement?: LicenseManagementUIPluginSetup;
+  maps?: MapsStartApi;
+  presentationUtil: PresentationUtilPluginStart;
   savedObjectsManagement: SavedObjectsManagementPluginStart;
   savedSearch: SavedSearchPublicPluginStart;
-  contentManagement: ContentManagementPublicStart;
-  presentationUtil: PresentationUtilPluginStart;
+  security?: SecurityPluginStart;
+  share: SharePluginStart;
+  spacesApi?: SpacesPluginStart;
+  triggersActionsUi?: TriggersAndActionsUIPublicPluginStart;
+  unifiedSearch: UnifiedSearchPublicPluginStart;
+  usageCollection?: UsageCollectionSetup;
+  uiActions: UiActionsStart;
```

Comment:
As above - this goes before `unifiedSearch` alphabetically.

Reply:
https://github.com/elastic/kibana/pull/170868/commits/c074629e08e42cdc0817800204f99196753632ee